accounts/abi/bind/bind.go

```diff
@@ -265,51 +345,69 @@ func Bind(types []string, abis []string, bytecodes []string, fsigs []map[string]
 			}
 		}
 	}
+
 	// Check if that type has already been identified as a library
 	for i := 0; i < len(types); i++ {
 		_, ok := isLib[types[i]]
 		contracts[types[i]].Library = ok
 	}
+
+	// compute the full set of libraries that each contract depends on.
+	for _, contract := range contracts {
+		if contract.Library {
+			continue
+		}
+		// recursively traverse the library dependency graph
+		// of the contract, flattening it into a set.
+		//
+		// For abigenv2, we do not generate contract deploy
+		// methods (which in v1 recursively deploy their
+		// library dependencies).  So, the entire set of
+		// library dependencies is required, and we will
```

Comment:
```suggestion
		// library dependencies is required, and we will store
```

Reply:
As this PR is written now: in the generated code, we store a full set of dependencies for each contract/libary (a list of `bind.MetaData` objects).  `v2.DeployContract` determines the deployment order at runtime, based on what's passed to it.

Reply:
So the word I forgot to add here is `determine` :)